### PR TITLE
Tlunizin 423

### PR DESCRIPTION
**Pull request by pushyamig**
Reviewers:@dlhaines @lsloan 
Optional:@bkirschn

### CODE REVIEW

**Comment by dlhaines** [comment]
I added a card for this review to Trello.  Nobody is assigned to it as yet.

https://trello.com/c/NEK1AqRP/21-tlunizin-423

File: sectionsUtilityTool/build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/sh --
```
Comment (lsloan) [inline sectionsUtilityTool/build.sh]: This script looks familiar.  :wink:

**Comment by lsloan** [comment]
:shipit:

**Comment by pushyamig** [comment]
@dlhaines can you merge to master?

**Pull request merged**